src/GafferImage/ImageSwitch.cpp

```diff
@@ -44,24 +44,15 @@ using namespace GafferImage;
 namespace
 {
 
-/// \todo If we introduce a TemporaryContext base class in Context.h
-/// then this should derive from that.
-struct ImageSwitchIndexContext
+struct ImageSwitchIndexContext : public Gaffer::Context::EditableScope
```

Comment:
I think we can ditch this class entirely now can't we, and just use `ImagePlug::GlobalScope` directly in SwitchTraits?